src/pcapi/scripts/booking/cancel_expired_bookings.py

```diff
@@ -0,0 +1,31 @@
+from datetime import datetime
+
+from pcapi.core.bookings import conf
+import pcapi.core.bookings.api as bookings_api
+from pcapi.core.bookings.repository import find_expiring_bookings
+from pcapi.utils.logger import logger
+
+
+def cancel_expired_bookings() -> None:
+    logger.info("[cancel_expired_bookings] Start")
+    bookings_to_expire = find_expiring_bookings()
+    bookings_in_error = []
+    canceled_bookings = []
+
+    for booking in bookings_to_expire:
+        try:
+            if datetime.utcnow() >= conf.CANCEL_EXPIRED_BOOKINGS_CRON_START_DATE:
+                bookings_api.cancel_expired_booking_by_system(booking)
+            canceled_bookings.append(booking.id)
+        except Exception as exc:
+            logger.exception("Encountered an exception: %s", exc)
```

Comment:
Je propose un message d'erreur plus précis :

```suggestion
            logger.exception("Could not cancel booking=%s: %s", booking.id, exc)
```

Reply:
J'ai un peu modifié ce qui est logé ici, mais les erreurs font apparaitre la représentation des `Bookings`, contenant leur `id`.